config/config.go

```diff
@@ -24,13 +24,13 @@ var (
 
 // GlobalConfig contains global config values
 type GlobalConfig struct {
-	DisableAnalytics         bool     `yaml:"disable_analytics"`
-	BuildkitCacheSizeMb      int      `yaml:"cache_size_mb"`
-	BuildkitImage            string   `yaml:"buildkit_image"`
-	DebuggerPort             int      `yaml:"debugger_port"`
-	BuildkitRestartTimeoutS  int      `yaml:"buildkit_restart_timeout_s"`
-	BuildkitAdditionalArgs   []string `yaml:"buildkit_additional_args"`
-	BuildkitAdditionalConfig string   `yaml:"buildkit_additional_config"`
+	DisableAnalytics         bool     `yaml:"disable_analytics" help:"Controls Earthly telemetry."`
+	BuildkitCacheSizeMb      int      `yaml:"cache_size_mb" help:"Size of the buildkit cache in Megabytes."`
+	BuildkitImage            string   `yaml:"buildkit_image" help:"Choose a specific image for your buildkitd."`
+	DebuggerPort             int      `yaml:"debugger_port" help:"What port should the debugger (and other interactive sessions) use to communicate."`
+	BuildkitRestartTimeoutS  int      `yaml:"buildkit_restart_timeout_s" help:"How long to wait for buildkit to (re)start, in seconds."`
+	BuildkitAdditionalArgs   []string `yaml:"buildkit_additional_args" help:"Additional args to pass to buildkit when it starts. Useful for custom certs, or user namespace complications."`
+	BuildkitAdditionalConfig string   `yaml:"buildkit_additional_config" help:"Additional config to use when starting the buildkit container; like mounting volumes."`
```

Comment:
Actually mounting volumes would be done through `BuilkditAdditionalArgs`. An example might be "using self-signed certificates"